src/main/kotlin/impl/GAuthImpl.kt

```diff
@@ -0,0 +1,177 @@
+package impl
+
+import GAuth
+import response.GAuthCode
+import response.GAuthToken
+import response.GAuthUserInfo
+import com.fasterxml.jackson.core.type.TypeReference
+import com.fasterxml.jackson.databind.ObjectMapper
+import enums.TokenType
+import exception.GAuthException
+import org.apache.http.HttpStatus
+import org.apache.http.client.methods.CloseableHttpResponse
+import org.apache.http.client.methods.HttpGet
+import org.apache.http.client.methods.HttpPatch
+import org.apache.http.client.methods.HttpPost
+import org.apache.http.client.methods.HttpRequestBase
+import org.apache.http.entity.StringEntity
+import org.apache.http.impl.client.CloseableHttpClient
+import org.apache.http.impl.client.HttpClientBuilder
+import org.json.simple.JSONObject
+import java.io.BufferedReader
+import java.io.InputStreamReader
+import java.nio.charset.StandardCharsets
+
+class GAuthImpl(
+    private val mapper: ObjectMapper,
+): GAuth {
```

Comment:
구현이니까
```suggestion
) : GAuth {
```
이렇게 띄어쓰기 하는게 좋을듯

Reply:
엇 이럴수가